index.ts

```diff
@@ -0,0 +1,26 @@
+import { AppSource } from "./util/database"
+import express from "express";
+import { insertMoviesData } from "./insertMoviesToDB";
+import{ Movie } from "./model/Movie" 
```

Comment:
Install https://prettier.io/. Always be careful with the esthetics of the code. 
import{ Movie } from "./model/Movie" ----->
import { Movie } from "./model/Movie";